File: bear.js
function bearDollars(arr) {
    let closeFriendDollars = 0;
    let acquaintanceDollars = 0;
    let friendDollars = 0;
    let regDollas = 0;
    for (let x of arr){
      let choice = x[1].toLowerCase();
      if (choice == "close friend"){
        closeFriendDollars += x[0] * 25;
      }else if  (choice == "acquaintance"){
        acquaintanceDollars += x[0] * 100;  
    }else if (choice == "friend"){
        friendDollars += x[0] * 50;  
    }else{
      regDollas += x[0] * 125;
    }
  }
    let moniesEarned = closeFriendDollars + acquaintanceDollars + friendDollars + regDollas;
    return moniesEarned;
    }

File: bear.py
def bear_dollars(jobs):
    close_friend_dollars = 0
    acquaintance_dollars = 0
    friend_dollars = 0
    reg_dollas = 0
    for x in range(0,len(jobs)):
        choice = str(jobs[x][1]).lower()
        if choice == "close friend":
            close_friend_dollars += jobs[x][0] * 25
        elif  choice == "acquaintance":
            acquaintance_dollars += jobs[x][0] * 100  
        elif choice == "friend":
            friend_dollars += jobs[x][0] * 50 
        else:
            reg_dollas += jobs[x][0] * 125
    monies_earned = close_friend_dollars + acquaintance_dollars + friend_dollars + reg_dollas
    return monies_earned